src/test/ui/consts/miri_unleashed/box.rs

```diff
@@ -0,0 +1,14 @@
+// compile-flags: -Zunleash-the-miri-inside-of-you
+#![feature(const_mut_refs, box_syntax)]
```

Comment:
I was quite surprised that I had to add `const_mut_refs` here, given that I unleashed miri. Is that expected?

Reply:
Yes. That was an explicit choice:
https://github.com/rust-lang/rust/blob/339a938fa6582d5c6f84d811680a1031c684c1c6/src/librustc_mir/transform/check_consts/validation.rs#L214-L216